tests/utils/fakes/fake_execution_collection.py

```diff
@@ -9,9 +10,10 @@
 
 class FakeDesignExecutionCollection(DesignExecutionCollection):
 
-    def trigger(self, execution_input: Score) -> DesignExecution:
+    def trigger(self, execution_input: Score, max_candidates: Optional[int] = None) -> DesignExecution:
         execution = DesignExecution()
         execution.score = execution_input
+        execution.max_candidates = max_candidates
```

Comment:
To remove `max_candidates` from the response, it would also be removed from here.